Python-Part4.ipynb

```diff
@@ -695,6 +790,25 @@
     "# Your code here\n",
     "filename = 'bird_observations.csv' #keep\n",
```

Comment:
Are these #keep comments relics of when we were planning on using a student convert script? Should they be deleted since we're not?